resources/views/pages/includes/header.blade.php

```diff
@@ -1,15 +1,15 @@
 @props([
     'title'        => null,
     'description'  => null,
-    'contentClass' => 'flex flex-col items-center text-center lg:space-x-16 contact-header',
+    'contentClass' => 'flex flex-col items-center text-center space-y-4 contact-header',
```

Comment:
this still results in a h1 having a bottom margin of 8px (0.5rem) and then an additional 16px (1rem) from the `space-y-4`, is that intended? Design seems to have only the 16px margin between the elements

Reply:
We'll put this down to user error